python/sglang/srt/managers/schedule_policy.py

```diff
@@ -77,18 +77,23 @@ def __init__(
         self,
         policy: str,
         tree_cache: BasePrefixCache,
-        enable_hierarchical_cache: bool = False,
     ):
         self.policy = self._validate_and_adjust_policy(policy, tree_cache)
         self.tree_cache = tree_cache
-        self.enable_hierarchical_cache = enable_hierarchical_cache
 
         # It is used to find the matching prefix for in-batch prefix caching.
         self.waiting_queue_radix_tree = RadixCache(
-            req_to_token_pool=None, token_to_kv_pool_allocator=None, disable=False
+            req_to_token_pool=None,
+            token_to_kv_pool_allocator=None,
+            page_size=1,
```

Comment:
Thank you for supporting page size >1!

I have a quick question here. Does it make sense to set `page_size=self.tree_cache.page_size` in this `waiting_queue_radix_tree`?  

Here we deprioritized the requests whose `len(in_batch_matching_prefixes)` is larger than `IN_BATCH_PREFIX_CACHING_DEPRIORITIZE_THRESHOLD` (32) to improve the cache hit rate. 

But using `page_size=1` may over estimate the matching length, which can incorrectly deprioritize requests, for example when `threshold < matching_length < page_size`.

Reply:
I do not think it will matter. This is because this `waiting_queue_radix_tree` and the whole logic of delayed scheduling all is based on heuristic and the `waiting_queue_radix_tree` does not precisely capture the real radix tree. so I will just use 1 here for simplicity.

In addition, this code path is unlikely to be used because we have changed our default scheduling policy from LPM to FCFS.